index.html

```diff
@@ -18,7 +18,11 @@
   <script>
     (function(window, videojs) {
       var player = window.player = videojs('videojs-vr-player');
-      player.vr({projection: '360', debug: true});
+      player.mediainfo = player.mediainfo || {};
```

Comment:
However, it seems to me like it might be possible to get rid of the `mediainfo` usage? Just do everything through the VR options and also expose it via `player.vr.projection`? Then update usage of current `mediainfo` to reflect that?